lighthouse-extension/app/manifest.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "__MSG_appName__",
-  "version": "1.6.0",
+  "version": "2.0.0.1",
```

Comment:
Yea CWS is annoying like this. I've done date releases in the past: `2019-05-09`.

The crx version doesn't have to correspond to the LH version. We have the latter in the report.